recipes/ckan/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "ckan" %}
+{% set version = "2.8.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/ckan/{{ name }}/archive/{{ name }}-{{ version }}.tar.gz
+  sha256: 1fdc09f4b1ae3a70d84138118c86eee056695235e5be8c42501c8af6b6679686
+
+build:
+  number: 0
+  skip: True  # [win]
+  entry_points:
+    - ckan = ckan.cli.cli:ckan
+  script: "{{ PYTHON }} -m pip install . -vv "
+
+requirements:
+  host:
+    - pip
+    - python <3.0
+    - setuptools
+  run:
+    - python <3.0
```

Comment:
I think there are a lot more requirements needed in order for CKAN to work, please look at https://github.com/ckan/ckan/blob/master/requirements.in

Reply:
Thanks, I will add these requirements.